pkg/frontend/clustermanager_putorpatch_test.go

```diff
@@ -226,3 +227,82 @@ func TestPutOrPatchClusterManagerConfiguration(t *testing.T) {
 		})
 	}
 }
+
+var populatedSystemData = api.SystemData{
+	LastModifiedAt:     &time.Time{},
+	LastModifiedBy:     "test-user",
+	LastModifiedByType: api.CreatedByTypeUser,
+	CreatedAt:          &time.Time{},
+	CreatedBy:          "test-user",
+	CreatedByType:      api.CreatedByTypeUser,
+}
+
+func TestEnrichSyncSetSystemData(t *testing.T) {
+	doc := api.ClusterManagerConfigurationDocument{
+		SyncSet: &api.SyncSet{},
+	}
+
+	enrichSyncSetSystemData(&doc, &populatedSystemData)
+
+	if err := compareSystemData(t, *doc.SyncSet.SystemData, populatedSystemData); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestEnrichSyncIdentityProviderSystemData(t *testing.T) {
+	doc := api.ClusterManagerConfigurationDocument{
+		SyncIdentityProvider: &api.SyncIdentityProvider{},
+	}
+
+	enrichSyncIdentityProviderSystemData(&doc, &populatedSystemData)
+
+	if err := compareSystemData(t, *doc.SyncIdentityProvider.SystemData, populatedSystemData); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestEnrichMachinePoolSystemData(t *testing.T) {
+	doc := api.ClusterManagerConfigurationDocument{
+		MachinePool: &api.MachinePool{},
+	}
+
+	enrichMachinePoolSystemData(&doc, &populatedSystemData)
+
+	if err := compareSystemData(t, *doc.MachinePool.SystemData, populatedSystemData); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestEnrichSecretSystemData(t *testing.T) {
+	doc := api.ClusterManagerConfigurationDocument{
+		Secret: &api.Secret{},
+	}
+
+	enrichSecretSystemData(&doc, &populatedSystemData)
+
+	if err := compareSystemData(t, *doc.Secret.SystemData, populatedSystemData); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func compareSystemData(t *testing.T, docSystemData, expectedSystemData api.SystemData) error {
```

Comment:
Do we still need `t`?
```suggestion
func compareSystemData(docSystemData, expectedSystemData api.SystemData) error {
```

Reply:
doh! I'll remove this, thanks.